README.md

```diff
@@ -19,6 +19,23 @@ Follow up instructions located [here](https://github.com/MoveOnOrg/Spoke/blob/ma
 
 Please let us know if you deployed by filling out this form [here](https://act.moveon.org/survey/tech/)
 
+## IMPORTANT
```

Comment:
move this to `docs/HOWTO_INTEGRATE_TWILIO.md`. Also don't make the migration required. I think it's fine to look up the campaign messaging service and fall back to the default logic if it isn't there.